main.go

```diff
@@ -109,3 +117,27 @@ func main() {
 		log.Fatal(err)
 	}
 }
+
+func cleanupPendingBuilds(curfs FileSystem) error {
+	l := log.New(os.Stderr, "[cleanup] ", log.LstdFlags)
+	projects, err := ioutil.ReadDir(cfg.ProjectsPath)
+	if err != nil {
+		return err
+	}
+
+	for _, f := range projects {
+		pendingPath := filepath.Join(cfg.BuildPath, f.Name(), "pending")
+		pendingBuilds, err := ioutil.ReadDir(pendingPath)
```

Comment:
How about just calling `os.RemoveAll()` and be done with it? `BootstrapProject()` will create the "pending" directory if it doesn't exist anyway.

Reply:
Nevermind my previous comment, we can't use `RemoveAll`, we must use `filesystem.Remove()`.

Reply:
Yep, the directory deletion here should depend on the file system.